widgets/post-carousel/post-carousel.php

```diff
@@ -100,19 +98,6 @@ function initialize() {
 		);
 	}
 
-	function enqueue_widget_scripts() {
-		wp_localize_script( 'sow-carousel-basic', 'carouselBreakpoints',
-			apply_filters(
-				'siteorigin_widgets_post_carousel_breakpoints',
```

Comment:
It's highly unlikely, but removing this filter might break some custom themes. Unless it's being added again somewhere else. Is there a way to have some backwards compat here? So still using this filter in `get_template_variables` and passing the results to the `$responsive_settings` variable.

Reply:
I could maybe add the filter here https://github.com/siteorigin/so-widgets-bundle/blob/67712369cf70ee5a7303565a7bff8381b58e86f5/widgets/post-carousel/post-carousel.php#L320 and rename any of the breakpoint settings to what they were - although that will make it harder to read the code.

Does that sound okay? If so, I'll make this change shortly.

Reply:
I'd suggest doing something more like this:

```
$breakpoint_defaults = apply_filters(
    'siteorigin_widgets_post_carousel_breakpoints',
    array(
        'tablet_landscape' => 1366,
		'tablet_portrait'  => 1025,
		'mobile'           => 480,
));
```

Then using these values as the defaults for the new array.

`'tablet_portrait_breakpoint' => ! empty ( $instance['responsive']['tablet']['portrait']['breakpoint'] ) ? $instance['responsive']['tablet']['portrait']['breakpoint'] : breakpoint_defaults['tablet_portrait'],`

Haven't thought it through fully, but this might do the trick?

Reply:
That's a better idea. Let's do that.